assembly/utils/Bytes.ts

```diff
@@ -55,49 +61,9 @@ export class Bytes {
         }
     }
 
-    static appendUint<T extends number> (b: Array<u8>, v: T, bitLength: number): void {
-        b.push(u8(v));
-        for (let i: u8 = 1; i < bitLength; i++) {
-            b.push(u8(v >> i * 8));
-        }
-    }
-
-    static encodeInteger (bytesBuffer: u8[], i: i32): i32 {
-        if (i < 1 << 6) {
-            Bytes.putUint<u8>(bytesBuffer, u8(i) << 2, BIT_LENGTH.INT_8);
-            return BIT_LENGTH.INT_8
-        }
-
-        if (i < 1 << 14) {
-            Bytes.putUint<u16>(bytesBuffer, u16(i << 2) + 1, BIT_LENGTH.INT_16);
-            return BIT_LENGTH.INT_16;
-        }
-
-        if (i < 1 << 30) {
-            Bytes.putUint<u32>(bytesBuffer, u32(i << 2) + 2, BIT_LENGTH.INT_32);
-            return BIT_LENGTH.INT_32;
-        }
-
-        const o = new Array<u8>(8);
-        let m = i;
-
-        let numBytes = 0;
-        for (; numBytes < 256 && m != 0; numBytes++) {
-            m = m >> 8;
-        }
-
-        const topSixBits: u8 = u8(numBytes - 4);
-        const lengthByte: u8 = topSixBits << 2 + 3;
-
-        Bytes.putUint<u8>(bytesBuffer, lengthByte, BIT_LENGTH.INT_8);
-        Bytes.putUint<u64>(o, i64(i), BIT_LENGTH.INT_64);
-
-        Bytes.copy<u8>(o.slice(0, numBytes), bytesBuffer, 1);
-
-        return numBytes + 1;
-    }
 
-    static decodeData (input: u8[]): DecodedData {
+    // Decrypt length from u8[] input
```

Comment:
- The term decrypt is not the correct one. It is rather `Decode`
- I think that a better naming of the function is `decodeCompactInt`